tests/functional/spa/jsonp.test.js

```diff
@@ -57,20 +56,19 @@ function runTest (title, htmlPage, supported) {
 }
 
 testDriver.test('JSONP on initial page load', supported, function (t, browser, router) {
-  t.plan(16)
 
   waitForPageLoad(browser, router, 'spa/jsonp/load.html')
     .then((result) => {
       let query, body
       ({ query, body } = result[1].request)
 
       let interactionTree = (body && body.length ? body : querypack.decode(query.e))[0]
+
       t.ok(interactionTree.end >= interactionTree.start, 'interaction end time should be >= start')
       t.ok(interactionTree.callbackEnd >= interactionTree.start, 'interaaction callback end should be >= interaction start')
       t.ok(interactionTree.callbackEnd <= interactionTree.end, 'interaction callback end should be <= interaction end')
-      t.equal(interactionTree.children.length, 1, 'expected one child node')
```

Comment:
Are the interactions getting extra nodes?

Reply:
Yea randomly the interaction will pick up and include an agent network call.